examples/basic/README.md

```diff
@@ -31,7 +31,11 @@ kubectl create ns nsm-system
 2. Apply NSM resources for basic tests:
 
 ```bash
-kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/examples/basic?ref=bbe46ecf104280e96ae3a32d4757149e792f0d11
+if [[ "${CALICO}" == "on" ]]; then
```

Comment:
Please don't use env. As I can see we can simply create a separate suite(setup) for calico and link at default use-cases.

Reply:
@denis-tingaikin ok, refactored example structure, please, take a look